app/views/user_mailer/in_person_failed_fraud.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= t('user_mailer.in_person_failed_suspected_fraud.greeting') %><br>
+<p>
+  <%= t(
+        'user_mailer.in_person_failed_suspected_fraud.body.intro',
+        location: @presenter.location_name,
+        date: @presenter.formatted_verified_date,
+      ) %>
```

Comment:
```suggestion
  <%= t(
        'user_mailer.in_person_failed_suspected_fraud.body.intro',
        app_name: APP_NAME,
        location: @presenter.location_name,
        date: @presenter.formatted_verified_date,
      ) %>
```
We should pass app_name here and use it instead of `Login.gov` in the translations